src/component/AddingForm.js

```diff
@@ -0,0 +1,27 @@
+import React, {Component} from 'react';
+
+
+class AddingForm extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+
+        };
+    }
+
+
```

Comment:
```suggestion
// no black here
```
just testing the suggestion code ...